src/store/groups.js

```diff
@@ -22,6 +22,13 @@
 
 const state = {
 	groups: [],
+	sortedGroups: [],
+	orderGroups: 'amount',
+}
+
+// the alphabetical sort is the default sort one so its undefined
+const sortFunc = {
+	amount: ({ contacts: a }, { contacts: b }) => b.length - a.length,
```

Comment:
This is an interesting syntax.
I find it hard to understand though, It's fine for destructuring assignments when it's necessary, but I would be more confident about using proper variables names.
```suggestion
	amount: (group1, group2) => group2.contacts.length - group1.contacts.length,
```
I don't think we lose any syntax quality here, but we définitive gain readability, no?